components/style/values/specified/counters.rs

```diff
@@ -65,3 +73,49 @@ fn parse_counters<'i, 't>(
         Err(input.new_custom_error(StyleParseErrorKind::UnspecifiedError))
     }
 }
+
+#[cfg(feature = "servo")]
+type CounterStyleType = ListStyleType;
+#[cfg(feature = "gecko")]
+type CounterStyleType = CounterStyleOrNone;
+
+/// The specified value for the `content` property.
+///
+/// https://drafts.csswg.org/css-content/#propdef-content
+#[derive(Clone, Debug, Eq, MallocSizeOf, PartialEq, ToComputedValue)]
+pub enum Content {
+    /// `normal` reserved keyword.
+    Normal,
+    /// `none` reserved keyword.
+    None,
+    /// `-moz-alt-content`.
+    #[cfg(feature = "gecko")]
+    MozAltContent,
+    /// Content items.
+    Items(Vec<ContentItem>),
+}
+
+/// Items for the `content` property.
+#[derive(Clone, Debug, Eq, MallocSizeOf, PartialEq, ToComputedValue)]
+pub enum ContentItem {
+    /// Literal string content.
+    String(String),
```

Comment:
And the `String`s around here should probably be changed to `Box<str>` in the same way. As I said, followup is fine for these.